### PR TITLE
add ordering for subjects

### PR DESCRIPTION
**Pull request by sewagodimo**
when a user joins/follows a subject they it will automatically move to the top of the subjects list. This will make navigation easier. Needs tests and it's super hacky

### CODE REVIEW

**Comment by sewagodimo** [comment]
@pjdelport please recommend a better way of doing this based on the decription

**Comment by sewagodimo** [comment]
I need to add more tests to this PR

File: src/buza/templates/buza/subject_list.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% extends 'base.html' %}
-{% load static %}
+{% load static buza_tags%}
```
Comment (PiDelport) [inline src/buza/templates/buza/subject_list.html]: I tend to try to keep template tag imports on separate lines, for easier management.

**Comment by PiDelport** [comment]
Discussed with @sewagodimo: we actually want to start a new branch for this.

**Pull request closed without merging**